apps/web/src/services/dashboards/widgets/compliance-check-status/ComplianceCheckStatusWidget.vue

```diff
@@ -216,77 +271,80 @@ const drawChart = (outerChartData, innerChartData) => {
     // outer
     const outerSeriesSettings = {
         categoryField: 'status',
-        valueField: 'rate',
+        valueField: 'value',
         startAngle: 160,
         endAngle: 380,
         radius: percent(250),
         innerRadius: percent(150),
     };
-    const outerSeries = createPieSeries(outerSeriesSettings);
-    chart.series.push(outerSeries);
+    const outerSeries = chartHelper.createPieSeries(outerSeriesSettings);
     outerSeries.labels.template.set('forceHidden', true);
-    outerSeries.ticks.template.set('visible', false);
+    outerSeries.ticks.template.set('forceHidden', true);
+    chart.series.push(outerSeries);
     outerSeries.slices.template.setAll({
-        toggleKey: 'none',
-        forceInactive: true,
         templateField: 'pieSettings',
         strokeOpacity: 0,
     });
+    const outerTooltip = chartHelper.createTooltip();
+    chartHelper.setPieTooltipText(outerSeries, outerTooltip);
+    outerSeries.slices.template.set('tooltip', outerTooltip);
     outerSeries.data.setAll(outerChartData);
 
     // inner
     const innerSeriesSettings = {
-        categoryField: 'status',
-        valueField: 'rate',
+        categoryField: 'severity',
+        valueField: 'value',
         startAngle: 160,
         endAngle: 380,
         radius: percent(100),
         innerRadius: percent(70),
     };
-    const innerSeries = createPieSeries(innerSeriesSettings);
+    const innerSeries = chartHelper.createPieSeries(innerSeriesSettings);
     chart.series.push(innerSeries);
     innerSeries.labels.template.set('forceHidden', true);
     innerSeries.ticks.template.set('visible', false);
     innerSeries.slices.template.setAll({
-        toggleKey: 'none',
-        forceInactive: true,
         templateField: 'pieSettings',
         strokeOpacity: 0,
     });
-
-    // TODO: replace with real data
-    setPieLabelText(chart, { text: '[fontSize:16px]Compliance score[/]:\n[fontSize:32px]45[/]' });
+    const innerTooltip = chartHelper.createTooltip();
+    chartHelper.setPieTooltipText(innerSeries, innerTooltip);
+    innerSeries.slices.template.set('tooltip', innerTooltip);
     innerSeries.data.setAll(innerChartData);
+
+    chartHelper.setPieLabelText(chart, { text: `[fontSize:16px]Compliance score[/]:\n[fontSize:32px]${state.score}[/]` });
 };
 
-const initWidget = async (data?: Data[]): Promise<Data[]> => {
+const initWidget = async (data?: Data): Promise<Data> => {
     state.loading = true;
     state.data = data ?? await fetchData();
+    if (!data) state.severityData = await fetchSeverityData();
```

Comment:
issue: When the user move from detail page to customize page, all the widgets are regenerated and use the data which is given as the parameter.
The thing is, at that time, `severityData` is not given.  So only the severity chart will not be drawn... 🥲